htsohm/plotting/data_points.py

```diff
@@ -0,0 +1,153 @@
+import os
+
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+import numpy as np
+from sqlalchemy import or_
+
+import htsohm
+from htsohm.db.__init__ import session, Material
```

Comment:
you never need to import from ___init__. the whole point of that file is that it is invisible.

from htsohm.db import session, Material